contracts/Manager.sol

```diff
@@ -167,10 +179,21 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
      * deployed and initialized.
      * @param _stakedCelo the address of the StakedCelo contract.
      * @param _account The address of the Account contract.
+     * @param _vote The address of the Vote contract.
      */
-    function setDependencies(address _stakedCelo, address _account) external onlyOwner {
+    function setDependencies(
```

Comment:
Calling this will have to be part of the upgrade process right? Anywhere we could capture this so we don't miss it?